### PR TITLE
goplus: build with "--build" instead of "--install"

### PR DESCRIPTION
**Pull request by cpunion**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/goplus.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Goplus < Formula
 
   def install
     ENV["GOPROOT_FINAL"] = libexec
-    system "go", "run", "cmd/make.go", "--install"
+    system "go", "run", "cmd/make.go", "--build"
```
Comment (carlocab) [inline Formula/goplus.rb:22]: What's the difference?

**Review by carlocab — COMMENTED** [review]
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

File: Formula/goplus.rb (hunk 1 above)
Comment (cpunion) [reply Formula/goplus.rb:22]: When install with "brew install goplus --build-from-source"，"--install" will create a link to $HOME/gop but "--build" won't.

Comment (carlocab) [reply Formula/goplus.rb:22]: It sounds like this isn't necessary then. Builds running inside `brew` cannot access the user's home directory by reading `$HOME`. On macOS, even if there was a way for the build to find out the user's home directory, the build cannot write to it due to the sandbox.

Comment (cpunion) [reply Formula/goplus.rb:22]: OK. Thanks for review.

**Pull request closed without merging**